ci/deps/travis-37-aarch64.yaml

```diff
@@ -0,0 +1,22 @@
+name: pandas-dev
```

Comment:
This file is not being used, can we remove it?

Reply:
Yes, We can remove it. I have replaced it with travis-37-arm64.yaml.